packages/terra-clinical-item-display/src/ItemComment.jsx

```diff
@@ -40,11 +47,12 @@ const ItemComment = ({
     ),
     customProps.className,
   );
+  const commentIcon = intl.formatMessage({ id: 'Terra.itemDisplay.comment' });
   return (
     <ItemDisplay
       text={text}
       isTruncated={isTruncated}
-      icon={<CommentIndicator className={cx('inline-icon')} />}
+      icon={<CommentIndicator a11yLabel={commentIcon} className={cx('inline-icon')} />}
```

Comment:
is this validated with Safari Voice-Over does the screen reader reads icon label text

Reply:
It does say empty group instead of image with earlier code which I have corrected now.  Tested in safari and chrome and it looks good.